### PR TITLE
Julia Solorzano content updates

### PR DESCRIPTION
**Pull request by juliaelman**
Updating some of my speaker content - thank you!

### CODE REVIEW

File: _presenters/julia-solorzano.md [hunk 1]
```diff
@@ -10,14 +10,16 @@ twitter: null
 website: null
 ---
 
-Julia Solórzano is the Security Experience Supervisor for [Login.gov](https://login.gov/) and is based in Durham, NC. She leads security, user experience and engineering teams, and coaches people to help them reach their full potential to create amazing things. Outside of Login.gov, her most recent work includes the [Department of Veterans API](https://developer.va.gov/) program and the [U.S. Web Design System](https://designsystem.digital.gov/). She is also a professor at Georgetown University where she teaches technology design and development.
+Julia Solórzano is the Security Experience Supervisor is based in Durham, NC. She leads security, user experience and engineering teams, and coaches people to help them reach their full potential to create amazing things. Outside of Login.gov, her most recent work includes [Login.gov](https://login.gov),  [Department of Veterans API](https://developer.va.gov/) program and the [U.S. Web Design System](https://designsystem.digital.gov/). She is also an adjunct professor at Georgetown University where she teaches technology design and development.
```
Comment (drewbrew) [inline _presenters/julia-solorzano.md:13]: It still reads as "Outside of Login.gov, her most recent work includes Login.gov." 

@juliaelman do you want to rephrase this?

**Review by sodevious — APPROVED** [review]

**Review by sodevious — CHANGES_REQUESTED** [review]
`Julia Solórzano is the Security Experience Supervisor based in Durham, NC` 

This may be a typo the way it's written with `the`

**Comment by drewbrew** [comment]
@sodevious It looks like @tim-schilling already fixed those wording issues. This one should be good to go after I fixed the merge conflicts.

**Review by drewbrew — APPROVED** [review]

**Review by jefftriplett — APPROVED** [review]

**Comment by jefftriplett** [comment]
@juliaelman thank you! 

We can follow up with a new PR if something needs to be corrected.

**Pull request merged**